apps/laboratory/tests/smart-account.spec.ts

```diff
@@ -4,7 +4,9 @@ import { EOA, SMART_ACCOUNT } from './shared/validators/ModalWalletValidator'
 
 import type { ModalWalletValidator } from './shared/validators/ModalWalletValidator'
 
-const NOT_ENABLED_EMAIL = 'test@w3ma.msdc.co'
+function getNotEnabledEmail(parallelIndex: number) {
+  return `test-${parallelIndex}@w3ma.msdc.co`
+}
```

Comment:
If we are able to check for arbitrary address, we can update `getEmailAddressToUse`  to accept optional secondary param that would override default domain with the other one

Reply:
done !